x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextField.java

```diff
@@ -8,11 +8,13 @@
 package org.elasticsearch.xpack.inference.mapper;
```

Comment:
Looks like a leftover? These changes are not needed.

Reply:
I don't understand, this is where we parse the mapping params?

Reply:
Done in 66d65b531846588a493dccd200e07af9f1daaf0b